.ci/ci.yml.tmpl

```diff
@@ -96,15 +111,15 @@ resources:
     - name: puppet-{{module}}-intermediate
       type: git-branch
       source:
-        uri: git@github.com:((github-account.username))/puppet-google-{{module}}.git
+        uri: git@github.com:((github-account.username))/{{submodule_name(module, puppet_custom_submodules, 'puppet-google-').strip()}}.git
         private_key: ((repo-key.private_key))
```

Comment:
probably too verbose - can we simplify it?

Reply:
Made this much simpler now that I'm not trying to generalize it.